### ISSUE TITLE
Revise conditions for defining "good" muon rings

### ISSUE THREAD

**Issue by moralejo** [cta-observatory/cta-lstchain#243]
https://github.com/cta-observatory/cta-lstchain/blob/4a8732d2d7789cf8af2a123e97153dcd0c424b92/lstchain/image/muon/muon_analysis.py#L220

There seems to be no condition checking whether the ring is actually a ring. The two events below from LST-1 run 01626 were labeled as good:

![Event_365_fitted](https://user-images.githubusercontent.com/14164282/70393832-d4200300-19ee-11ea-99cc-b93b1a9b38a2.png)
![Event_343_fitted](https://user-images.githubusercontent.com/14164282/70393837-de420180-19ee-11ea-905e-491e31d78894.png)

**Comment by rlopezcoto** [comment]
I agree that these are obviously not "good" muons.
The conditions for good muons refer to width, radius, impact, etc... We discussed about trying to get a goodness of fit out of minuit but I did not manage to get it, so events like the one above, that are tagged as "good" muons, can be eliminated if we apply additional conditions:

- ring completeness. For the analysis I showed, ring completeness > 0.9, meaning that there has to be 90% of the ring illuminated above a given threshold. For the two rings above you get most of the ring not illuminated and therefore they do not enter into the analysis
- even selecting only the ones that are very much complete, there can be very large hadrons that fulfill this condition. I then apply also the condition that the size outside of the ring has to be < 100 phe. Meaning that if we have hadrons illuminating several parts of the camera, they will produce a lot of light also outside the ring.

I'll work on re-defining what "good" muons are
